Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/xarray-tutorial/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/usr/share/miniconda3/envs/xarray-tutorial/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/share/miniconda3/envs/xarray-tutorial/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/share/miniconda3/envs/xarray-tutorial/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/share/miniconda3/envs/xarray-tutorial/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/usr/share/miniconda3/envs/xarray-tutorial/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/usr/share/miniconda3/envs/xarray-tutorial/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/share/miniconda3/envs/xarray-tutorial/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# to be able to read unit attributes following the CF conventions
import cf_xarray.units  # must be imported before pint_xarray
import pint_xarray

xr.set_options(display_expand_data=False)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Input [0;32mIn [3][0m, in [0;36m<cell line: 2>[0;34m()[0m
[1;32m      1[0m [38;5;66;03m# to be able to read unit attributes following the CF conventions[39;00m
[0;32m----> 2[0m [38;5;28;01mimport[39;00m [38;5;21;01mcf_xarray[39;00m[38;5;21;01m.[39;00m[38;5;21;01munits[39;00m  [38;5;66;03m# must be imported before pint_xarray[39;00m
[1;32m      3[0m [38;5;28;01mimport[39;00m [38;5;21;01mpint_xarray[39;00m
[1;32m      5[0m xr[38;5;241m.[39mset_options(display_expand_data[38;5;241m=[39m[38;5;28;01mFalse[39;00m)

File [0;32m/usr/share/miniconda3/envs/xarray-tutorial/lib/python3.10/site-packages/cf_xarray/units.py:6[0m, in [0;36m<module>[0;34m[0m
[1;32m      3[0m [38;5;28;01mimport[39;00m [38;5;21;01mre[39;00m
[1;32m      4[0m [38;5;28;01mimport[39;00m [38;5;21;01mwarnings[39;00m
[0;32m----> 6[0m [38;5;28;01mimport[39;00m [38;5;21;01mpint[39;00m
[1;32m      7[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpint[39;00m [38;5;28;01mimport[39;00m (  [38;5;66;03m# noqa: F401[39;00m
[1;32m      8[0m     DimensionalityError,
[1;32m      9[0m     UndefinedUnitError,
[1;32m     10[0m     UnitStrippedWarning,
[1;32m     11[0m )
[1;32m     13[0m [38;5;66;03m# from `xclim`'s unit support module with permission of the maintainers[39;00m

[0;31mModuleNotFoundError[0m: No module named 'pint'
ModuleNotFoundError: No module named 'pint'

